test/reflection.jl

```diff
@@ -213,3 +213,10 @@ let t13464 = "hey there sailor"
         @test startswith(err13464.msg, "expression is not a function call, or is too complex")
     end
 end
+
+# Issue 14173
+module Tmp
+    export A
+    A = randn(2000, 2000)
+end
+@test @elapsed(whos(IOBuffer(), Tmp)) < 5
```

Comment:
Also probably better to rename `Tmp` to something like `Tmp14173` to avoid unnecessary conflict when adding test (to the same file) in the future.